independent-publisher-connectors/Krozu PM (Independent Publisher)/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "9830a1ef-36f5-490b-956b-5732ac2c2cd2",
```

Comment:
@GodChosen secret look alike values are not allowed. please replace with dummy value. you can use "Enter your client ID".

Reply:
@Amjed-Ayoub Replaced with a dummy value.